_rules/auto-update-text-efbfc7.md

```diff
@@ -292,21 +288,10 @@ This `span` element contains text content that is automatically **changed** but
 </body>
 ```
 
-#### Inapplicable Example 5
-
-This `span` element with text content that automatically changes multiple times is **alone** in the document.
-
-```html
-<body onload="startUpdates()">
-	<span id="target">1</span>
-
-	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
-</body>
-```
```

Comment:
I'm a little bit confused where this inapplicable example came from and couldn't find anything in the understanding docs. I would personally fail this example, so I removed it for the moment, re-adding it would mean changing the new applicability.

Reply:
Is it worth adding it to the failed examples? - It is a hard call and might be worth asking @Jym77 or @carlosapaduarte for their thoughts?

Reply:
[SC 2.2.2](https://www.w3.org/TR/WCAG21/#pause-stop-hide) states (emphasises mine):
> For moving, [blinking](https://www.w3.org/TR/WCAG21/#dfn-blinking), scrolling, or auto-updating information, all of the following are true:
> * Moving, blinking, scrolling: For any moving, blinking or scrolling information that (1) starts automatically, (2) lasts more than five seconds, and **(3) is presented in parallel with other content**, there is a mechanism for the user to pause, stop, or hide it unless the movement, blinking, or scrolling is part of an activity where it is essential; and
> * Auto-updating: For any auto-updating information that (1) starts automatically and **(2) is presented in parallel with other content**, there is a mechanism for the user to pause, stop, or hide it or to control the frequency of the update unless the auto-updating is part of an activity where it is essential.

This content is not "presented in parallel with other content" and therefore shouldn't fail the SC.